www/js/incident/post-trip-prompt.js

```diff
@@ -119,7 +119,20 @@ angular.module('emission.incident.posttrip.prompt', ['emission.plugin.logger'])
   var displayCompletedTrip = function(notification, state, data) {
       $rootScope.displayingIncident = true;
       Logger.log("About to display completed trip from Notification");
-      $state.go("root.main.incident", notification.data);
+      $state.go("root.main.enketosurvey", {
+        form_location: "json/trip-end-survey_v6.json",
+            opts: JSON.stringify({
+              session: {
+                data_key: 'manual/confirm_survey',
+                trip_properties: {
+                  // TODO: use the right variable
+                  start_ts: notification.data.properties.start_ts,
```

Comment:
@atton16 I thought you were now passing in the trip object directly instead of the start and end timestamps?

Reply:
Ahh yes, I get it now why you are referring to `$rootScope.displayingIncident`. But even though I called `readAndUpdateForDay`, how am I supposed to know which trip the notification is referring to?